locales/en-US/app.ftl

```diff
@@ -177,13 +177,18 @@ startedDateLabel = Experiment start date: <b>{$startedDate}</b>
 nonExperimentDialogHeaderLink = Test Pilot
 
 ## Label shown next to a series of icons indicating whether an experiment is available as an add-on, mobile app, and/or web site
-experimentPlatformWebAddonMobile = Firefox / web / mobile experiment
-experimentPlatformWebAddon = Firefox / web experiment
-experimentPlatformWebMobile = web / mobile experiment
-experimentPlatformAddonMobile = Firefox / mobile experiment
-experimentPlatformWeb = web experiment
 experimentPlatformAddon = Firefox experiment
-experimentPlatformMobileApp = mobile experiment
+experimentPlatformAndroid = Android experiment
+experimentPlatformIos = iOS experiment
+experimentPlatformWeb = web experiment
+experimentPlatformAddonWeb = Firefox / web experiment
+experimentPlatformAddonAndroid = Android / Firefox experiment
+experimentPlatformAddonIos = iOS / Firefox experiment
+experimentPlatformAddonAndroidIos = Android / iOS / Firefox experiment
+experimentPlatformAddonAndroidWeb = Android / Firefox / web experiment
+experimentPlatformAddonAndroidIosWeb = Android / iOS / Firefox / web experiment
+experimentPlatformAndroidWeb = Android / web experiment
+experimentPlatformAndroidIos = Android / iOS experiment
```

Comment:
I'd need to chart this out to be sure of all possible combinations (if we care right now), but i think we're missing at least: `"iOS / web experiment"`